cargo-miri/src/phases.rs

```diff
@@ -541,6 +541,27 @@ pub fn phase_runner(mut binary_args: impl Iterator<Item = String>, phase: Runner
     }
 }
 
+fn get_miriflags() -> Option<Vec<String>> {
```

Comment:
Why the `Option<Vec>` return type? We can just use an empty `Vec` to reflect absence of any specified flags.